examples/build-package/main.tf

```diff
@@ -63,6 +63,45 @@ module "package_dir_poetry" {
   artifacts_dir = "${path.root}/builds/package_dir_poetry/"
 }
 
+# Create zip-archive of a src directory where "poetry export" & "pip install --no-deps" will also be executed (using docker)
+module "package_src_poetry" {
```

Comment:
This example generates a ZIP archive containing the following:

```
# terraform apply -target module.package_src_poetry
...
module.package_src_poetry.null_resource.archive[0] (local-exec): Created: ./builds/package_src_poetry/c3749e24b5ed617a9f16837ae948730e548f4327270af01d9b0761560687d23f.zip
module.package_src_poetry.null_resource.archive[0]: Creation complete after 4s [id=937883823961125271]

# unzip -t ./builds/package_src_poetry/c3749e24b5ed617a9f16837ae948730e548f4327270af01d9b0761560687d23f.zip
Archive:  ./builds/package_src_poetry/c3749e24b5ed617a9f16837ae948730e548f4327270af01d9b0761560687d23f.zip
    testing: python39_app_src_poetry/   OK
    testing: python39_app_src_poetry/__init__.py   OK
    testing: requirements.txt         OK
    testing: colorful/                OK
    testing: colorful/__init__.py     OK
    testing: colorful/ansi.py         OK
    testing: colorful/colors.py       OK
    testing: colorful/core.py         OK
    testing: colorful/styles.py       OK
    testing: colorful/terminal.py     OK
    testing: colorful/utils.py        OK
    testing: colorful/data/           OK
    testing: colorful/data/colornames.json   OK
    testing: colorful/data/rgb.txt    OK
    testing: colorful-0.5.5.dist-info/   OK
    testing: colorful-0.5.5.dist-info/INSTALLER   OK
    testing: colorful-0.5.5.dist-info/LICENSE   OK
    testing: colorful-0.5.5.dist-info/METADATA   OK
    testing: colorful-0.5.5.dist-info/RECORD   OK
    testing: colorful-0.5.5.dist-info/REQUESTED   OK
    testing: colorful-0.5.5.dist-info/WHEEL   OK
    testing: colorful-0.5.5.dist-info/top_level.txt   OK
    testing: pyproject.toml           OK
No errors detected in compressed data of ./builds/package_src_poetry/c3749e24b5ed617a9f16837ae948730e548f4327270af01d9b0761560687d23f.zip.
```

Reply:
Based on the changes I made in this PR, this is my expected output.

Which file(s) are you expecting or not expecting to be a part of the ZIP?

Reply:
There is no `__init__.py` in the `src` directory, by design when using the `src` directory structure.

Reference: 
* https://packaging.python.org/en/latest/discussions/src-layout-vs-flat-layout/
* https://python-poetry.org/docs/cli/#new

Reply:
Oh, I did not look at the fixture's content, `python3.9-app-src-poetry/src` does contain a module named `python39_app_src_poetry`, my bad.